### PR TITLE
feat(README): add detailed description of Docker commands

### LINKED ISSUE #5: describe docker run commands in more detail
e.g. as found in https://github.com/music-encoding/music-encoding.github.io/blob/main/CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by musicEnfanthen**
This PR adds descriptions of the Docker commands used in the README.

Fixes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,15 +9,35 @@ The image contains all the necessary dependencies:
...
+
+
+### Pulling the Docker image
 
 ```bash
 docker pull ghcr.io/music-encoding/docker-mei:latest
 ```
 
-## Example usage
+Alternatively, the image will be automatically pulled when you run a docker container via `docker run`.
```
Comment (bwbohl) [inline README.md:40]: Maybe, add something that the image will not be automatically updated to the latest version if you have run it before? So it is advisable to run a `docker pull` from time to time.

File: README.md [hunk 2]
```diff
@@ -29,7 +49,7 @@ docker run --rm -v /ABSOLUTE/PATH/TO/YOUR/MUSIC_ENCODING/CLONE:/opt/docker-mei/m
 For example, if you start the Docker image from the root folder of your music-encoding repository clone:
 
 ```bash
-docker run --rm -v $(pwd):/opt/docker-mei/music-encoding --name docker-mei ghcr.io/music-encoding/docker-mei:latest ant -noinput -buildfile music-encoding/build.xml -Ddocker=true
+docker run --rm -v `pwd`:/opt/docker-mei/music-encoding --name docker-mei ghcr.io/music-encoding/docker-mei:latest ant -noinput -buildfile music-encoding/build.xml -Ddocker=true
```
[the remark this answers is not in the document]
Comment (musicEnfanthen) [reply README.md:52]: I was just harmonizing the two occurences. And, at least, ` 'pwd' ` seems to work on Windows.

File: README.md (hunk 1 above)
Comment (musicEnfanthen) [reply README.md:40]: Good point

Comment (musicEnfanthen) [inline README.md:40]:
```suggestion
Although the image will be automatically pulled when you run a docker container via `docker run`, the image will not be automatically updated to the latest version on subsequent runs. So it is recommended to run the `docker pull` command above from time to time.
```

Comment (musicEnfanthen) [reply README.md:40]: @bwbohl Like this?

**Pull request merged**